cuprum/unittests/test_cqrs_runtime_behaviour.py

```
@@ -0,0 +1,420 @@
+    catalogue, python_program = python_catalogue()
+    python = sh.make(python_program, catalogue=catalogue)
+    producer = python(
+        "-c",
+        "\n".join(
+            (
+                "import sys",
+                "for _ in range(4096):",
+                "    sys.stdout.write('payload-line\\n')",
+                "    sys.stdout.flush()",
+            ),
+        ),
```

Ruff (0.15.20)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)